libs/barista-components/autocomplete/src/autocomplete-trigger.ts

```diff
@@ -270,6 +284,9 @@ export class DtAutocompleteTrigger<T>
     @Optional()
     @Inject(DT_UI_TEST_CONFIG)
     private _config?: DtUiTestConfiguration,
+    @Optional()
+    @Inject(DT_OPTION_CONFIG)
+    _optionConfig?: DtOptionConfiguration,
```

Comment:
NIT: if it is not a property in the class please don't prefix with an underscore (for consistency). We only prefix properties that are internal or private and accessible from outside with an underscore.

```suggestion
    optionConfig?: DtOptionConfiguration,
```